tests/src/test/java/alluxio/shell/command/ChownCommandTest.java

```diff
@@ -39,6 +45,60 @@ public void chown() throws IOException, AlluxioException {
     Assert.assertEquals("user2", owner);
   }
 
+  @Test
+  public void chownValidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String newOwner = "alice";
+    List<String> groups = CommonUtils.getGroups(newOwner);
+    String group = groups.get(0);
+    String[] command = new String[] {"chown", newOwner + ":" + group, "/testFile"};
+    String expectedCommandOutput =
+        "Changed owner:group of /testFile to alice:" +  group + ".";
+    runChownOwnerAndGroup(command, 0, expectedCommandOutput, newOwner, group);
+  }
+
+  @Test
+  public void chownInvalidOwnerValidGroup() throws Exception {
+    setupTestGroupMappingService();
+    FileSystemTestUtils.createByteFile(mFileSystem, "/testFile", WriteType.MUST_CACHE, 10);
+    String user = "alice";
+    String nonexistUser = "nonexistuser";
+    List<String> groups = CommonUtils.getGroups(user);
+    String originalOwner = mFileSystem.getStatus(new AlluxioURI("/testFile")).getOwner();
+    String originalGroup = mFileSystem.getStatus(new AlluxioURI("/testFile")).getGroup();
+    String group = groups.get(0);
+    String[] command = new String[] {"chown", nonexistUser + ":" + group, "/testFile"};
+    String expectedCommandOutput = "Could not setOwner for /testFile.";
```

Comment:
Currently, Alluxio allows the admin to change the ownership of a path to arbitrary owner by doing: chown <newOwner> <path>. Whether the owner is valid or not is not checked because in this case Alluxio does not attempt to acquire the group information of this owner. This may need some further discussion, but I think maybe it's beyond the scope of this PR. Furthermore, currently the shell will print out some message saying the owner does not belong to group, which will prompt the admin to check the username and group name.